docs/manifest.json

```diff
@@ -519,6 +519,12 @@
 									"title": "Workspace Scheduling",
 									"description": "Learn how to control how workspaces are started and stopped",
 									"path": "./admin/templates/managing-templates/schedule.md"
+								},
+								{
+									"title": "Early Access External Workspaces",
```

Comment:
We don't need to put "early access" in the title if we have it as state

Reply:
Good catch. I was getting confused where it is coming from.